database/postgres/dal/news.py

```diff
@@ -27,3 +27,19 @@ def fetch_news(self, order_by=None, limit=None, filters=None):
     def get_news_by_id(self, blob_id: str):
         return self.session.query(News).filter(News.blob_id == blob_id).first()
 
+
+    def get_news_by_real_id(self, news_id: str):
+        return self.session.query(News).filter(News.news_id == news_id).first()
+
+
+    def get_sport_by_name(self, sport_name):
+        return self.session.query(Sport).filter_by(sport_name = sport_name).first()
+
+
+    def get_news_ids(self, time_limit):
```

Comment:
I see `get_news_ids` but I don't understand what news? What ids? Judging by the argument of the method we expect some time limit (which is also a very vague name).

`get_news_by_time_limit` for your case, but 'time_limit' is a vague name, so specify what exactly does it mean.